scss/_patterns_status.scss

```diff
@@ -0,0 +1,61 @@
+@import 'settings';
+
+// Default status button styling
+@mixin vf-p-status {
+  @include vf-p-button-validated;
+  @include vf-p-button-in-progress;
+  @include vf-p-button-new;
+  @include vf-p-button-updated;
+  @include vf-p-button-deprecated;
+}
+
+%p-button--status {
+  border-radius: 0.125rem;
+  font-size: 1rem;
+  line-height: 1.5rem;
+  padding: calc(#{$spv-nudge - $sp-unit * 0.5} - 1px) $sph-inner--small * 1.5;
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+}
+
+// Override local variables
+@mixin vf-p-button-validated {
+  .p-button--validated {
+    @extend %p-button--status;
+    background-color: #006b75;
```

Comment:
You need set hex values to a variable before using. I would suggest setting the vars at the top of the main mixin (vf-p-status) so its easy to change in the future.